### PR TITLE
added analytics to Stem 22-1995 form review screen

### PR DESCRIPTION
**Pull request by ksantiagoBAH**
## Description
Implement accordion expand / collapse tracking on Form 1995 Summary page
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/7462
## Testing done
Local testing

## Screenshots
Screen Shot 2020-06-11 at 12 50 41 PM

## Acceptance criteria
- [x] Analytics are added to 1995 review accordions

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by sonntag-adam — APPROVED** [review]

File: src/platform/forms-system/src/js/review/ReviewChapters.jsx [hunk 1]
```diff
@@ -32,8 +33,18 @@ class ReviewChapters extends React.Component {
   handleToggleChapter({ name, open, pageKeys }) {
     if (open) {
       this.props.closeReviewChapter(name, pageKeys);
+      if (this.props.form.formId === '22-1995') {
+        recordEvent({
+          event: 'nav-accordion-collapse',
+        });
+      }
```
Comment (f1337) [inline src/platform/forms-system/src/js/review/ReviewChapters.jsx:40]: I'm new here and don't have a ton of context yet. Please pardon me if the following is not helpful. :)

Not a blocker, but a question & possible suggestion: Is this a pattern we expect to repeat elsewhere? If so, would it make future work faster, cleaner, and/or easier if we encapsulated this logic in a method that could be invoked like:

`this.props.recordFormEvent(formId, 'nav-accordion-collapse);`

**Review by f1337 — APPROVED** [review]
LGTM (looks good to me!). One non blocking comment/suggestion.

File: src/platform/forms-system/src/js/review/ReviewChapters.jsx [hunk 2]
```diff
@@ -32,8 +33,18 @@ class ReviewChapters extends React.Component {
   handleToggleChapter({ name, open, pageKeys }) {
     if (open) {
       this.props.closeReviewChapter(name, pageKeys);
+      if (this.props.form.formId === '22-1995') {
```
Comment (cvalarida) [inline src/platform/forms-system/src/js/review/ReviewChapters.jsx:36]: Hard-coding logic for a specific form isn't an approach we can accept into the forms library. Please find another way to accomplish this. I recommend either looking into callbacks we can pass from the form config, or having a conversation with the analytics team to determine if this is useful information to gather for all forms.

**Review by cvalarida — CHANGES_REQUESTED** [review]

**Comment by ksantiagomorgan6** [comment]
OBE

**Pull request closed without merging**